src/Catty/DataModel/Project/Project.h

```diff
@@ -53,15 +47,10 @@
 - (void)translateDefaultProject;
 - (void)renameToProjectName:(NSString* _Nonnull)projectName andShowSaveNotification:(BOOL)showSaveNotification;
 - (void)renameToProjectName:(NSString* _Nonnull)projectName andProjectId:(NSString* _Nonnull)projectId andShowSaveNotification:(BOOL)showSaveNotification;
-- (void)renameObject:(SpriteObject* _Nonnull)object toName:(NSString* _Nonnull)newObjectName;
 - (void)updateDescriptionWithText:(NSString* _Nonnull)descriptionText;
-- (nonnull NSArray*)allObjectNames;
-- (NSArray<SpriteObject*>* _Nonnull)allObjects;
```

Comment:
Please do not remove this method. Currently, it should return all objects of the single Scene (later it should return all objects of all Scenes).